packages/formstr-app/src/containers/CreateForm/FormBuilder.tsx

```diff
@@ -14,8 +15,17 @@ function FormBuilder() {
   const {
     isRightSettingsOpen,
     closeOnOutsideClick: closeSettingsOnOutsideClick,
+    setFormTempId,
+    formTempId,
   } = useFormBuilderContext();
 
+  useEffect(() => {
+    if (formTempId === "") {
+      const id = makeTag(6);
+      setFormTempId(id);
+    }
+  });
```

Comment:
You need to specify a dependency array. If you don't specify it then the hook will run everytime. Here, an empty array needs to be specified

Reply:
why empty? formId and setFormId are dependencies right?